### PR TITLE
feat: Confirmation when initiating a call in a large group(SQSERVICES-1923)

### PR DESCRIPTION
**Pull request by arjita-mitra**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQSERVICES-1923" title="SQSERVICES-1923" target="_blank">StorySQSERVICES-1923</a>  Prevent accidental calls in large groups  [Web] Confirmation when initiating a call in a large group
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - Prevent accidental calls in large groups [Web] Confirmation when initiating a call in a large group

- The **PR Description**
  - As a Wire user, I would like to see a confirmation screen when calling a large group so that I do not disturb a large group of people inadvertently
----
##### References
1. https://wearezeta.atlassian.net/browse/SQSERVICES-1923

### CODE REVIEW

File: src/i18n/en-US.json [hunk 1]
```diff
@@ -254,6 +254,9 @@
   "callingRestrictedConferenceCallPersonalModalTitle": "Feature unavailable",
   "callingRestrictedConferenceCallTeamMemberModalDescription": "To start a conference call, your team needs to upgrade to the Enterprise plan.",
   "callingRestrictedConferenceCallTeamMemberModalTitle": "Feature unavailable",
+  "groupCallConfirmationModalTitle": "Are you sure you want to call {{groupSize}} people",
```
Comment (PatrykBuniX) [inline src/i18n/en-US.json]: Don't we need question mark at the end here?

File: src/script/view_model/CallingViewModel.ts [hunk 2]
```diff
@@ -88,7 +88,7 @@ declare global {
     setSinkId?: (sinkId: string) => Promise<void>;
   }
 }
-
+const maxGroupSize = 4;
```
Comment (PatrykBuniX) [inline src/script/view_model/CallingViewModel.ts:91]: Is it really 4? 😅

File: src/script/view_model/CallingViewModel.ts [hunk 3]
```diff
@@ -374,7 +374,25 @@ export class CallingViewModel {
...
+              preventClose: true,
+              primaryAction: {
+                action: async () => await startCall(conversationEntity, CALL_TYPE.NORMAL),
+                text: t('groupCallModalPrimaryBtnName'),
+              },
+              secondaryAction: {
+                text: t('modalConfirmSecondary'),
+              },
+              text: {
+                message: t('groupCallConfirmationModalTitle', maxGroupSize),
```
Comment (V-Gira) [inline src/script/view_model/CallingViewModel.ts]: Shouldn't this be `memberCount` ?

```suggestion
                message: t('groupCallConfirmationModalTitle', memberCount),
```

File: src/script/view_model/CallingViewModel.ts (hunk 2 above)
Comment (arjita-mitra) [reply src/script/view_model/CallingViewModel.ts:91]: Yes. Currently on iOS we show a dialog when a user is initiating a call with more than 4 people and we need to implement the same for Web.

https://wearezeta.atlassian.net/browse/SQSERVICES-1923

**Review by V-Gira — APPROVED** [review]

**Pull request merged**